includes/admin/settings/register-settings.php

```diff
@@ -2205,65 +2216,63 @@ function edd_sendwp_callback($args) {
-	$connect_url = $activated ? edd_jilt()->get_connect_url() : '';
-	$account_url = $connected ? edd_jilt()->get_integration()->get_jilt_app_url() : '';
+function edd_recapture_callback($args) {
+	$client_connected = false;
 
-	echo wp_kses_post( $args['desc'] );
+	if ( class_exists( 'RecaptureEDD' ) ) {
+		$client_connected = RecaptureEDD::is_ready();
+	}
 
-	if ( $activated ) :
-		?>
+	ob_start();
 
-		<?php if ( $connected ) : ?>
+	echo $args['desc'];
 
+	// Output the appropriate button and label based on connection status
+	if ( $client_connected ) :
+		$connection_comlpete = get_option('recapture_api_key');
+		?>
+		<div class="inline notice notice-<?php echo $connection_comlpete ? 'success' : 'warning'; ?>">
 			<p>
-				<button id="edd-jilt-disconnect" class="button"><?php esc_html_e( 'Disconnect Jilt', 'easy-digital-downloads' ); ?></button>
+				<?php _e( 'Recapture plugin activated.', 'easy-digital-downloads' ); ?>
+				<?php printf( __( '%sAccess your Recapture account%s.', 'easy-digital-downloads' ), '<a href="https://recapture.io/account" target="_blank" rel="noopener noreferrer">', '</a>' ); ?>
 			</p>
 
+			<?php if ( $connection_comlpete ) : ?>
 			<p>
-				<?php
-				wp_kses_post(
-					sprintf(
-						/* Translators: %1$s - <a> tag, %2$s - </a> tag */
-						__( '%1$sClick here%2$s to visit your Jilt dashboard', 'easy-digital-downloads' ),
-						'<a href="' . esc_url( $account_url ) . '" target="_blank">',
-						'</a>'
-					)
-				);
-				?>
+				<a id="edd-recapture-disconnect" class="button" href="<?php echo esc_html( admin_url( 'admin.php?page=recapture-confirm-disconnect' ) ); ?>"><?php _e( 'Disconnect Recapture', 'easy-digital-downloads' ); ?></a>
 			</p>
-
-		<?php else : ?>
-
+			<?php else : ?>
 			<p>
-				<a id="edd-jilt-connect" class="button button-primary" href="<?php echo esc_url( $connect_url ); ?>">
-					<?php esc_html_e( 'Connect to Jilt', 'easy-digital-downloads' ); ?>
-				</a>
+				<?php printf( __( 'Complete your connection to Recapture %shere%s.', 'easy-digital-downloads' ), '<a href="' . admin_url( 'admin.php?page=recapture' ) . '">', '</a>' ); ?>
 			</p>
-
-		<?php endif; ?>
-
-	<?php elseif( current_user_can( 'install_plugins' ) ) : ?>
-
+			<?php endif; ?>
+		</div>
+	<?php
+	else :
+		?>
+		<p>
+			<?php _e( 'We recommend Recapture for recovering lost revenue by automatically sending effective, targeted emails to customers who abandon their shopping cart.', 'easy-digital-downloads' ); ?>
+		</p>
+		<p>
+			<?php printf( __( '%sLearn More%s (Free trial available)' ), '<a href="https://recapture.io/abandoned-carts-easy-digital-downloads" taregt="_blank" rel="noopener noreferrer">', '</a>' ); ?>
+		</p>
 		<p>
-			<button id="edd-jilt-connect" class="button button-primary">
-				<?php esc_html_e( 'Install Jilt', 'easy-digital-downloads' ); ?>
+			<button type="button" id="edd-recapture-connect" class="button button-primary"><?php esc_html_e( 'Connect with Recapture', 'easy-digital-downloads' ); ?>
```

Comment:
We should only show this button if `current_user_can( 'install_plugins' )`. IMO we could probably show the above "We recommend Recapture..." text, just shouldn't output the button.

Reply:
:+1: